cmd/cored/main.go

```diff
@@ -143,6 +147,55 @@ func runServer() {
 	ctx := context.Background()
 	env.Parse()
 
+	raftDir := filepath.Join(*dir, "raft") // TODO(kr): better name for this
```

Comment:
Isn't this `DIR/raft` now? But maybe we should add the raft id or something. (Not sure what makes a good directory name...)